crates/actor/wit/deps/wasmcloud/bus.wit

```diff
@@ -1,49 +1,20 @@
 package wasmcloud:bus;
 
 interface lattice {
-    /// Interface target. This represents an interface, which can be selected by `set-target`.
+    /// Interface target. This represents an interface, which can be selected by `set-link-name`.
     /// The set of `target-*` functions defines all "selectable" interfaces provided by the host.
     /// Implementations of `wasmcloud:bus/lattice` may extend the set of "selectable" interfaces.
-    resource target-interface {
-        constructor(namespace: string, %package: string, %interface: string);
-
-        /// `wasi:blobstore/blobstore` interface target
-        wasi-blobstore-blobstore: static func() -> target-interface;
-
-        /// `wasi:keyvalue/atomic` interface target
-        wasi-keyvalue-atomic: static func() -> target-interface;
-
-        /// `wasi:keyvalue/eventual` interface target
-        wasi-keyvalue-eventual: static func() -> target-interface;
-
-        /// `wasi:logging/logging` interface target
-        wasi-logging-logging: static func() -> target-interface;
-
-        /// `wasmcloud:messaging/consumer` interface target
-        wasmcloud-messaging-consumer: static func() -> target-interface;
-
-        /// `wasi:http/outgoing-handler` interface target
-        wasi-http-outgoing-handler: static func() -> target-interface;
-    }
-
-    /// Actor identifier
-    variant actor-identifier {
-        /// Actor public key
-        public-key(string),
-        /// Actor call alias
-        alias(string),
+    resource call-target-interface {
+        constructor(namespace: string, %package: string, %interface: string, %function: option<string>);
     }
 
-    /// Target entity
-    variant target-entity {
-        /// Link target paired with an optional link name
-        link(option<string>),
-        /// Actor target
-        actor(actor-identifier),
-    }
+    /// Set an optional target for all interfaces specified. If `target` is `none`, then target is set to default.
+    /// todo(vados-cosmonic): the list of interfaces should not be necessary once signing is removed
```

Comment:
I think that we should not remove interface list here - developers should be able to set a link name for a "batch" of interfaces in a single call